openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/GlossaryTermRepository.java

```diff
@@ -699,6 +714,11 @@ protected void updateTags(
         return; // Nothing to update
       }
 
+      // If equal return
+      if (validateIfTagsAreEqual(origTags, updatedTags)) {
```

Comment:
can we add inverse check here and not have returns in between a method

Reply:
Good Point , updated @harshach thanks